src/shared/capture/capturev4l.h

```diff
@@ -0,0 +1,350 @@
+//========================================================================
+//  This software is free: you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License Version 3,
+//  as published by the Free Software Foundation.
+//
+//  This software is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  Version 3 in the file COPYING that came with this distribution.
+//  If not, see <http://www.gnu.org/licenses/>.
+//
+//========================================================================
+// Some portions of software interfacing with XIO originally writtten
+// by James Bruce as part of the CMVision library.
+//      http://www.cs.cmu.edu/~jbruce/cmvision/
+// (C) 2004-2006 James R. Bruce, Carnegie Mellon University
+// Licenced under the GNU General Public License (GPL) version 2,
+//   or alternately by a specific written agreement.
+//========================================================================
+// (C) 2016 Eric Zavesky, Emoters
+// Licenced under the GNU General Public License (GPL) version 3,
+//   or alternately by a specific written agreement.
+//========================================================================
+/*!
+ \file    capturev4l.h
+ \brief   C++ Interface: CaptureV4L
+ \author  Eric Zavesky, (C) 2016 (derived from DC1394 and CMVision)
+ */
+//========================================================================
+
+#ifndef CAPTUREV4l_H
+#define CAPTUREV4l_H
+#include "captureinterface.h"
+#include "util.h"
+#include <sys/time.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string>
+#include "VarTypes.h"
+#include <linux/videodev2.h>
+#include <sys/poll.h>
+
+#include <map>
+
+//#include "conversions.h"
```

Comment:
Not sure what is incorrect about this commit.  Was it the commented-out "conversions.h" include?